sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -48,31 +56,35 @@ template <typename Group, size_t Extent> class group_with_scratchpad {
 };
 
 // ---- sorters
-template <typename Compare = std::less<>> class default_sorter {
-  Compare comp;
+namespace default_sorters {
+
+template <typename CompareT = std::less<>> class joint_sorter {
+  CompareT comp;
   std::byte *scratch;
   size_t scratch_size;
 
 public:
   template <size_t Extent>
-  default_sorter(sycl::span<std::byte, Extent> scratch_,
-                 Compare comp_ = Compare())
+  joint_sorter(sycl::span<std::byte, Extent> scratch_,
+               CompareT comp_ = CompareT())
       : comp(comp_), scratch(scratch_.data()), scratch_size(scratch_.size()) {}
 
   template <typename Group, typename Ptr>
   void operator()(Group g, Ptr first, Ptr last) {
+    (void)g;
+    (void)first;
+    (void)last;
```

Comment:
Could you please explain why it's moved from the `#else` branch here?

Reply:
Good point. They are moved to the #else branch.